reports/m1_proposal.md

```diff
@@ -4,6 +4,13 @@ Rachel Bouwer, Koray Tecimer, He Ma, Yimeng Xia
 
 ## Motivation and Purpose
 
+Our role: Data scientist consultancy firm
+
+Target audience: Record Companies
+
+One of the biggest sectors in the entertainment industry in terms of market size is music. It was valued at USD 27,898.47 million in 2022 according to Industry Research Co. In this huge economy record companies can strategize in a data driven way to make smart investments to artists. But it might not be clear which artists are worth investing in. To address this issue we will create a dashboard application that will be easy to browse thorough artists and genres. Record labels will be able to see which factors play into account for artists and songs to be popular. Our app will allow them to identify past songs and evaluate current artists, but also give them ideas on which artist they should invest next. Our app will allow users to filter artists and genres and show plots that will allow them to compare artists and also factors that make that artist popular or unpopular.
```

Comment:
```suggestion
One of the biggest sectors in the entertainment industry in terms of market size is music. It was valued at USD 27,898.47 million in 2022 according to Industry Research Co. In this huge economy, record companies can strategize in a data driven way to make smart investments to artists - but it might not be clear which artists are worth investing in. To address this issue, we will create a dashboard application that will be easy to browse thorough artists and genres. Record labels will be able to see which factors play into account for artists and songs to be popular. Our app will allow them to identify past songs and evaluate current artists, but also give them ideas on which artist they should invest next. Our app will allow users to filter artists and genres and show plots that will allow them to compare artists and also factors that make that artist popular or unpopular.
```